server.js

```diff
@@ -164,7 +164,14 @@ app.use('/tools', require('./controllers/tools'));
 app.use('/patterns', require('./controllers/pattern-library'));
 
 /**
- * Handle archived paths first:
+ * Homepage handler
+ * Mount as a one-off GET route in both English and Welsh
+ */
+app.get('/', require('./controllers/home'));
```

Comment:
Would this be better as:

`app.get(['/', '/welsh'], require('./controllers/home'))`?

(eg. avoiding the duplication)

Or is it better explicit like here?

Reply:
Possibly, but that array syntax via `app.use` caused the original but so opted to be explicit.